views/pulsar/v2/helpers/nav.html.twig

```diff
@@ -56,12 +56,12 @@
 {% endmacro %}
 
 
-{% macro nav_item(options) %}
+{% macro nav_item(options, index) %}
 {% spaceless %}
     {% import '@pulsar/pulsar/v2/helpers/html.html.twig' as html %}
     {% import '@pulsar/pulsar/v2/helpers/util.html.twig' as util %}
 
-    <li
+    <li id="nav-item-{{ index }}"
```

Comment:
As far as I am aware we have only one instance of the Primary Navigation every time. No, we don't need a specific namespace but just giving it an id like "01" felt not descriptive enough.